markets/perps-market/contracts/utils/OffchainUtil.sol

```diff
@@ -56,6 +66,20 @@ library OffchainUtil {
         );
 
         IPythVerifier.PriceFeed memory pythData = priceFeeds[0];
+
+        // Double check publishTime is valid
+        if (
+            pythData.price.publishTime < asyncOrder.settlementTime ||
+            pythData.price.publishTime >
+            asyncOrder.settlementTime + settlementStrategy.priceWindowDuration
+        ) {
+            revert PythPriceOutsideSettlementWindow(
```

Comment:
Me and david chatted about this for l1perps. I was saying that if we cant trust pyth to check validity window how can we trust that the publish time is correct?